dataland-backend/src/main/kotlin/org/dataland/datalandbackend/controller/DataController.kt

```diff
@@ -31,17 +32,18 @@ abstract class DataController<T>(
     var dataMetaInformationManager: DataMetaInformationManager,
     var objectMapper: ObjectMapper,
     private val clazz: Class<T>,
+    @Autowired private val permissionChecker: PermissionChecker,
```

Comment:
Überprüft einmal ob man das Autowired hier braucht. Der PermissionChecker lebt ja wie der DataMetaInformationManager auch im Backend, könnte sein, dass man das hier ähnlich nicht braucht.

Reply:
ok ich teste ohne autowired

Reply:
I will try this at the very end after everything is good to go. So that if it breaks things, i don't have to rerun the toolbox to get everything back to normal

Reply:
ok